File: app/src/main/java/com/omvp/app/ui/samples/social/view/SampleSocialView.java
package com.omvp.app.ui.samples.social.view;

import com.omvp.app.base.mvp.view.BaseView;

/**
 * Created by Angel on 21/02/2018.
 */
public interface SampleSocialView extends BaseView {
    void drawImage(String imageUrl);

    void drawName(String name);

    void drawEmail(String email);

    void showHideSocialButtons(boolean show);
}

File: app/src/main/java/com/omvp/app/ui/samples/social/view/SampleSocialView.kt
package com.omvp.app.ui.samples.social.view

import com.omvp.app.base.mvp.view.BaseView

interface SampleSocialView : BaseView {

    fun drawImage (imageUrl: String)

    fun drawName(name: String)

    fun drawEmail(email: String)

    fun showHideSocialButtons(show: Boolean)
}
